sources/functions/deluge

```diff
@@ -3,11 +3,6 @@
 function whiptail_deluge () {
   if [[ -z $deluge ]]; then
     case $(lsb_release -cs) in
```

Comment:
The `case .. in` seems a bit abundant now

Reply:
I may just leave this considering I may need to reuse it in the future when building 1.3.15 no longer becomes viable (i.e. Bullseye and beyond)